packages/lib-content/src/translations/i18n.js

```diff
@@ -0,0 +1,33 @@
+import i18n from 'i18next'
+import {
+  initReactI18next,
+  useTranslation as useBaseTranslation
+} from 'react-i18next'
+
+const libI18n = i18n.createInstance()
+libI18n.use(initReactI18next).init({
+  fallbackLng: 'en',
+  interpolation: {
+    escapeValue: false // not needed for react as it escapes by default
+  },
+  react: {
+    useSuspense: false
+  }
+})
+
+libI18n.addResourceBundle('en', 'translation', require('./en.json'))
```

Comment:
Only English for now until all of lib-content is built with Get Involved, Home pages, etc